event_model/__init__.py

```diff
@@ -1953,15 +2037,15 @@ def compose_stream_resource(
 
 def compose_stop(
     *,
-    start,
-    event_counters,
-    poison_pill,
-    exit_status="success",
-    reason="",
-    uid=None,
-    time=None,
-    validate=True,
-):
+    start: dict,
```

Comment:
```suggestion
    start: RunStart,
```
etc for the other compose functions